prometheus/converters.go

```diff
@@ -0,0 +1,170 @@
+package prometheus
```

Comment:
Maybe? I didn't put it under there because there wasn't an associated service running. I think of things under `services` as things that have network listeners.

Reply:
That's a good point.